pom.xml

```diff
@@ -20,4 +20,19 @@
         <module>zip</module>
         <module>java</module>
     </modules>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <junit.version>4.13.1</junit.version>
+        <xalan.version>2.7.2</xalan.version>
+        <xmlsec.version>2.3.0</xmlsec.version>
+        <httpclient.version>4.5.13</httpclient.version>
+        <bouncycastle.version>1.61</bouncycastle.version>
```

Comment:
1.67?

Reply:
It was on a old commit. I've updated it.